src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlBinaryWriter.cs

```diff
@@ -303,9 +303,9 @@ public override void WriteEndAttribute()
             _textNodeOffset = -1;
         }
 
-        public override void WriteXmlnsAttribute(string prefix, string ns)
+        public override void WriteXmlnsAttribute(string? prefix, string ns)
```

Comment:
`public override void WriteStartAttribute(string prefix, string localName)` might be worth filing a bug or at least TODO-NULLABLE

Reply:
The `WriteStartAttribute` methods are `internal` and are never called with `null` for `prefix`, so I left them as non-nullable and didn't change the code. But `WriteXmlnsAttribute` could be called with null, so that's why I fixed it here.